src/application-manager.coffee

```diff
@@ -732,26 +699,33 @@ module.exports = class ApplicationManager extends EventEmitter
 					hostnameOnHost
 				}
 				_.assign(configOpts, opts)
+
 				volumes = JSON.parse(app.volumes)
-				volumes = _.mapValues volumes, (volumeConfig) ->
+				volumes = _.mapValues volumes, (volumeConfig, volumeName) =>
 					volumeConfig ?= {}
 					volumeConfig.labels ?= {}
-					return volumeConfig
+					@createTargetVolume(volumeName, app.appId, volumeConfig)
+
+				networks = JSON.parse(app.networks)
+				networks = _.mapValues networks, (networkConfig, networkName) =>
+					networkConfig ? {}
```

Comment:
I think you meant `networkConfig ?=  {}`? In which case you could define the default above as `networks = _.mapValues networks, (networkConfig = {}, networkName) =>`

Reply:
How can `networkConfig` be null or undefined?

Reply:
We can't be certain of any of the values that come in from docker-compose files, as users can do anything like to add them to the api. I'm going to add runtime validation with io-ts but for now, we use checks like this.